packages/wdio-browserstack-service/src/scripts/accessibility-scripts.ts

```diff
@@ -0,0 +1,69 @@
+import path from 'node:path'
+import fs from 'node:fs'
+import os from 'node:os'
+
+class AccessibilityScripts {
+    private static instance: AccessibilityScripts | null = null
+
+    public performScan: string | null = null
+    public getResults: string | null = null
+    public getResultsSummary: string | null = null
+    public saveTestResults: string | null = null
+    public commandsToWrap: Array<any> | null = null
+
+    public browserstackFolderPath = path.join(os.homedir(), '.browserstack')
+    public commandsPath = path.join(this.browserstackFolderPath, 'commands.json')
+
+    constructor() {}
+
```

Comment:
```suggestion
```

Reply:
or with a comment as to why we do this
```suggestion
    // don't allow to create instances from it other than through `checkAndGetInstance`
    private constructor() {}

```